src/boss/cloudvm/local_worker.go

```diff
@@ -0,0 +1,99 @@
+package cloudvm
+
+import (
+	"fmt"
+	"log"
+	"net/http"
+	"os"
+	"os/exec"
+	"path/filepath"
+
+	"github.com/open-lambda/open-lambda/ol/common"
+)
+
+// WORKER IMPLEMENTATION: LocalWorker
+type LocalWorkerPoolPlatform struct {
+	// no platform specific attributes
+}
+
+func NewLocalWorkerPool() *WorkerPool {
+	return &WorkerPool{
+		WorkerPoolPlatform: &LocalWorkerPoolPlatform{},
+	}
+}
+
+func (_ *LocalWorkerPoolPlatform) NewWorker(workerId string) *Worker {
+	return &Worker{
+		workerId: workerId,
+		workerIp: "",
+	}
+}
+
+func (_ *LocalWorkerPoolPlatform) CreateInstance(worker *Worker) {
```

Comment:
Maybe we should leave it to a separate PR (and have a TODO for now), but it seems like CreateInstance and DeleteInstance should be returning errors instead of panic'ing or suppressing them.